src/cpp/include/openvino/genai/image_generation/image2image_pipeline.hpp

```diff
@@ -49,6 +49,14 @@ class OPENVINO_GENAI_EXPORTS Image2ImagePipeline {
         const UNet2DConditionModel& unet,
         const AutoencoderKL& vae);
 
+    // creates Flux pipeline from building blocks
+    static Image2ImagePipeline flux(
```

Comment:
please, add python bindings similar to others:
https://github.com/openvinotoolkit/openvino.genai/blob/af41f9c68fe787f25fcbf12dc83e81b69a839a1c/src/python/py_image_generation_pipelines.cpp#L330-L332